ompi/communicator/communicator.h

```diff
@@ -233,7 +234,7 @@ typedef struct ompi_communicator_t ompi_communicator_t;
  * This is done so we don't end up needing a structure that is
  * incredibly larger than necessary because of the bitness.
  */
-#define PREDEFINED_COMMUNICATOR_PAD (sizeof(void*) * 192)
+#define PREDEFINED_COMMUNICATOR_PAD (sizeof(void*) * 194)
```

Comment:
That's a fair point. I can add the following language to the comment:
```
To keep the ompi_predefined_communicator_t structure stable in size padding is
added to it beyond the size of the ompi_communicator_t. For example, If a single
pointer is added to ompi_communicator_t then the PREDEFINED_COMMUNICATOR_PAD
must by increased by 1 to keep the size of the ompi_predefined_communicator_t
structure the same.
```

What do you think?